include/pops/config.hpp

```diff
@@ -70,6 +72,11 @@ class Config
     bool use_mortality{false};
     double mortality_rate{0};
     int first_mortality_year{0};  // TODO: document that it starts at 1, not 0
+    unsigned last_index{0};
+    bool use_movements{false};
+    std::vector<std::vector<int>> movements;
```

Comment:
I think `movements` should be a parameter of run_step() function like the data are and the things like Treatments. `last_index` should be part of the Module class because it is something what is changing during simulation.

The current design of movement is not perfect, I think for the future we may try to come up with something like Treatments class for the movements so that things like last_index are encapsulated there and run_step itself is more flexible.

Reply:
Yes, I agree with Vashek on the movement class needing to be changed long-term. I have added #87 for that as a future improvement.